homeassistant/components/enigma2/config_flow.py

```diff
@@ -55,6 +65,35 @@
 )
 
 
+async def get_options_schema(handler: SchemaCommonFlowHandler) -> vol.Schema:
+    """Get the options schema."""
+    hass = handler.parent_handler.hass
+    entry = cast(SchemaOptionsFlowHandler, handler.parent_handler).config_entry
+    device: OpenWebIfDevice = hass.data[DOMAIN][entry.entry_id]
+    bouquets = [b[1] for b in (await device.get_all_bouquets())["bouquets"]]
+
+    return vol.Schema(
+        {
+            vol.Optional(CONF_DEEP_STANDBY): selector.BooleanSelector(),
+            vol.Optional(CONF_SOURCE_BOUQUET): selector.SelectSelector(
+                selector.SelectSelectorConfig(
+                    options=bouquets,
+                    multiple=False,
+                    custom_value=True,
```

Comment:
The bouquets are retrieved from the device; why allow custom values here?

Reply:
```suggestion
```
Yes, you are right.